db/template.go

```diff
@@ -14,7 +14,14 @@ import (
 
 // CreateTemplate creates a new workflow template
 func (d TinkDB) CreateTemplate(ctx context.Context, name string, data string, id uuid.UUID) error {
-	_, err := wflow.Parse([]byte(data))
```

Comment:
Why are you doing this?! If the template is not valid we should not go further. Can you revert or explain why you moved this down there?

Reply:
Yes you are right. I moved it down by mistake. Will fix it.

Reply:
Done